### PR TITLE
13122 oracle optimisation

### PR DESCRIPTION
**Pull request by zlakomanoff**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #13122 

Hint for speedup foreghn key query on the 11g oracle (12g no speedup, but and no slowdown) + improved sql formatting.
Improved oci tests.

### CODE REVIEW

**Review by dynasource — APPROVED** [review]
The markup looks a lot better. Couldn't find the optimization part though.

**Comment by zlakomanoff** [comment]
https://github.com/yiisoft/yii2/pull/13175/commits/582e1de425fe3d2574c97fc07356e0efaa50a65a#diff-a73229348903931d4a5c6de05b4eb6f5R263

File: tests/data/config.php [hunk 1]
```diff
@@ -43,6 +43,12 @@
             'password' => 'postgres',
             'fixture' => __DIR__ . '/postgres.sql',
         ],
+        'oci' => [
```
Comment (samdark) [inline tests/data/config.php]: @cebe will it run in travis?

**Review by samdark — APPROVED** [review]

File: tests/data/config.php (hunk 1 above)
Comment (cebe) [inline tests/data/config.php]: no, phpunit on travis runs with --exclude=oci,...

**Review by SilverFire — APPROVED** [review]

**Comment by SilverFire** [comment]
Merged: https://github.com/yiisoft/yii2/commit/2645b5ac780a047438af933d0ee3b45ff9441ea6
Thank you!

**Pull request closed without merging**